Formula/dafny.rb

```diff
@@ -33,7 +33,7 @@ def install
 
     (bin/"dafny").write <<~EOS
       #!/bin/bash
-      exec "#{Formula["dotnet@6"].opt_bin}/dotnet" "#{libexec}/Dafny.dll" "$@"
+      exec dotnet "#{libexec}/Dafny.dll" "$@"
```

Comment:
```suggestion
      exec "#{Formula["dotnet"].opt_bin}/dotnet" "#{libexec}/Dafny.dll" "$@"
```

Reply:
This needs to be set to make sure we are pointing at the correct `dotnet`.

Reply:
Makes sense, fixed.